dev_guide/port_forwarding.adoc

```diff
@@ -12,6 +12,13 @@
 toc::[]
 
 == Overview
+
+{product-title} takes advantage of a feature built into
```

Comment:
Looks like this sentence is same as the one in `architecture/additional_concepts/port_forwarding.adoc` please make sure these are consistent.